src/redux/actions/samples/createSamples.js

```diff
@@ -116,15 +113,14 @@ const createSamples = (
         createdDate,
         lastModified: createdDate,
         options,
-        metadata: experiment?.metadataKeys
-          .reduce((acc, curr) => ({ ...acc, [curr]: METADATA_DEFAULT_VALUE }), {}) || {},
-        files: Object.values(files).reduce(((acc, curr) => {
-          const fileType = fileNameForApiV1[getFileTypeV2(curr.name, sampleTechnology)];
```

Comment:
The whole concept of file name as it was being used has been removed (it was being used as a file type in disguise). Now we use only the `sampleFileTypes` (these match the values in the `sample_file_type` enum in sql).

The only file name that still remains is the **actual** file's name which is in `fileObject` but is not used by us for anything other than to determine the file type at the beginning. In this there's no longer a mix of ways to classify files. `fileObject` is not created by us directly, but when reading the original file in `loadAndCompressIfNecessary`

Reply:
you say the actual file name is not used anymore but in file `src/components/data-management/FileUploadModal.jsx` you readded this code (same that was used before)
```
  const getFilePathToDisplay = (fileObject) => (
    // if the file has a path, trim to just the file and its folder.
    // otherwise simply use its name
    fileObject.path
      ? _.trim(Object.values(getFileSampleAndName(fileObject.path)).join('/'), '/')
      : fileObject.name
```
can this be removed then? or did you find some compatibility issues?